test/e2e/runner/main.go

```diff
@@ -334,7 +299,11 @@ Does not run any perturbations.
 				return err
 			}
 
-			return Cleanup(cli.testnet)
+			if err := Cleanup(cli.testnet); err != nil {
+				return err
+			}
+
+			return nil
```

Comment:
[nit]
isn't this the same? or something weird was happenig?

Reply:
nah just copy pasted from upstream. seems like we can simplify

Reply:
this was just copy pasted from upstream, I'm not sure why it was like that